src/main/java/gov/usgs/wma/statistics/model/JsonDataBuilder.java

```diff
@@ -320,6 +326,16 @@ protected JsonDataBuilder intermediateValue(Value sample) {
 	}
 	
 	public JsonDataBuilder intermediateValues(List<? extends Value> samples) {
+		if(samples == null || samples.isEmpty()) {
+			return this;
+		}
+		String month = samples.get(0).getMonth();
+		LOGGER.trace("month add request {} ", month);
+		if (this.intermediateMonthsAdded .contains(month)) {
+			return this;
+		}
+		LOGGER.trace("month add perform {}", month);
+		this.intermediateMonthsAdded.add(month);
```

Comment:
Here is the code to avoid added a month twice.

Reply:
Cool!